### ISSUE TITLE
Links

### ISSUE THREAD

**Issue by overath2** [noQ-sweden/noq#620]
**Story**
As a onboarding person i need to access our links so i can easily read more about this in my browser.

**Acceptance Criteria (checklist)**
- [ ]  All links autmatically available in browser day 1 for onboarding person if possilble else
- [ ]  All links can be imported manually

**Other info:** 
[Ex: Links, Solutions, best practices etc.
](https://docs.google.com/document/d/1ou6wr1sI9IJZaGGSMn_SJvFru5FJQKMDRZY16kBmx5Y/edit#heading=h.2et92p0)

- [ ]  **Definition of Ready OK?**
Also called DoR, our generic process for checking if this is ready for start. See our Playbook here for details: https://docs.google.com/document/d/1ou6wr1sI9IJZaGGSMn_SJvFru5FJQKMDRZY16kBmx5Y/edit#heading=h.3rdcrjn

- [ ]  **Definition of Done OK?**
Also called DoD, our generic process for checking this is done in the right way. See our Playbook here for details: https://docs.google.com/document/d/1ou6wr1sI9IJZaGGSMn_SJvFru5FJQKMDRZY16kBmx5Y/edit#heading=h.23ckvvd

**Comment by patrik-pwned** [comment]
Är denna fortfarande aktuell?

@overath2 @jespon508

**Comment by overath2** [comment]
Ja. Sist i vår playbook är en massa länkar som förut funkade. Jag skulle vilja ha dessa testade och infogade i ett dokument så man kan importera till sin browser enjelt.

**Comment by amirnoq** [comment]
We could create a bookmark file with instructions to import it to the browser. After importing it you will see all the links in your bookmarks. Maybe even categorized into different folders. What do you think?

**Comment by overath2** [comment]
This is what i call proactiveness @amirnoq ! Great idea, prio it with you PO or/and team coach.

**Comment by overath2** [comment]
Great idea!
Can you create a sheet with all these links so i can verify them first?

**Comment by amirnoq** [comment]
I'm using these links that you mentioned in the ticket: 

> [Ex: Links, Solutions, best practices etc.](https://docs.google.com/document/d/1ou6wr1sI9IJZaGGSMn_SJvFru5FJQKMDRZY16kBmx5Y/edit#heading=h.2et92p0)

**Comment by overath2** [comment]
I have fixed all links here: https://docs.google.com/spreadsheets/d/1wJBZYpoezRHQUQXuV9J2faA2kptqfpR8zjyKD8ojl-I/edit?usp=sharing

**Comment by jespon508** [comment]
Hi,

This is good work. I felt the same after onboarding.

Best Jessica

Den sön 23 feb. 2025 kl 15:05 skrev Ove Holmberg (noQ) <
***@***.***>:

**Comment by overath2** [comment]
@amirnoq will add instructions to playbook here:
https://docs.google.com/document/d/1nbKJWM_FspoZzlSmbCFTnHwqOSiT1lfsvzozmtopduY/edit?tab=t.0#heading=h.2et92p0
Then notify on Slack for all noQ to check this link
Ove will add the link above to onbaording checklist.

**Comment by amirnoq** [comment]
I've updated the [playbook](https://docs.google.com/document/d/1nbKJWM_FspoZzlSmbCFTnHwqOSiT1lfsvzozmtopduY/edit?tab=t.0#heading=h.2et92p0) and made a gist to store the bookmark file here: https://gist.github.com/amirnoq/1a183d5c2bd8e4ec34c850cdc9ab0e1d

I tried my best to add everything so If something is wrong or missing you comment on the [gist](https://gist.github.com/amirnoq/1a183d5c2bd8e4ec34c850cdc9ab0e1d) or message me on slack.
